app/components/NavModal/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+const NavModalDialog = styled.div.attrs({
+  className: 'modal-dialog',
+})`
+  &&& {
+    font-size: 2rem;
+    font-weight: 600;
+  }
+`;
+
+const NavModalContent = styled.div.attrs({
+  className: 'modal-content',
+})`
+  &&& {
+    background: transparent;
+    border: transparent;
+
+    & > div {
+      border: inherit;
+    }
+  }
+`;
+
+const ModalCloseX = styled.span`
+  color: #ddd;
+  font-size: 3.25rem;
+  text-shadow: none;
+`;
+
+const ModalLink = styled(Link)`
+  &&& {
+    color: #b8daff;
+  }
+`;
+
+function NavModal() {
+  // Callback for click event: Modal needs help closing properly after clicking on link (otherwise overlay will stay and make site unusable).
+  function hideNavModal() {
+    document.getElementById('nav-modal-close-btn').click();
+  }
+
+  return (
+    <div className="modal fade" id="nav-modal" role="dialog" aria-hidden="true">
+      <NavModalDialog role="document">
+        <NavModalContent>
+          <div className="modal-header" data-dismiss="modal">
+            <button type="button" className="close" id="nav-modal-close-btn" aria-label="Close">
+              <ModalCloseX aria-hidden="true">&times;</ModalCloseX>
+            </button>
+          </div>
+          <div className="modal-body">
+            <ul className="nav flex-column text-center" onClick={hideNavModal}>
+              <li className="nav-item">
+                <ModalLink className="nav-link active" to="/">Home</ModalLink>
```

Comment:
Any ideas for how to handle which is actually active on a per-page basis? Each page can pass in some prop that keys to one of these.

Reply:
I'm unsure. I don't think I'm familiar enough with things to have any ideas yet. I figured the nav would just list all the available front-facing pages without regard to where the user currently is (I mean, that's what I'd expect from a navigation popup, but I don't browse a lot on my phone).

Reply:
We can probably remove the "active" from the classNames then, not sure if that affects styling